docs/tutorials/model_to_instance.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 数理モデルからインスタンスを生成\n",
+    "\n",
+    "JijModelingで数理モデルを記述した後、ソルバーで解くには、その数理モデルが持つPlaceholderにデータを代入する必要があります。この操作を実現するのが `Interpreter.eval_problem` メソッドです。このページでは、ナップサック問題を例に、OMMX Adapter経由でソルバーに解かせる方法を説明します。\n",
+    "\n",
+    "ナップサック問題を定式化して得られる数理モデルは次の通りです：\n",
+    "\n",
+    "$$\n",
+    "\\begin{align*}\n",
+    "\\mathrm{maximize} \\quad & \\sum_{i=0}^{N-1} v_i x_i \\\\\n",
+    "\\mathrm{s.t.} \\quad & \\sum_{i=0}^{n-1} w_i x_i \\leq W, \\\\\n",
+    "& x_{i} \\in \\{ 0, 1\\} \n",
+    "\\end{align*}\n",
+    "$$\n",
+    "\n",
+    ":::{note}\n",
+    "ナップサック問題の定式化について詳しく知りたい場合は [こちら](https://www.documentation.jijzept.com/docs/tutorial/knapsack/) を参照してください。\n",
+    ":::\n",
+    "\n",
+    "この数理モデルにあるそれぞれの文字の意味は以下の通りです：\n",
+    "\n",
+    "| 文字 | 説明 |\n",
+    "| --- | --- |\n",
+    "| $N$ |\tアイテムの総数 |\n",
+    "| $v_{i}$ | アイテム $i$ の価値 |\n",
+    "| $w_{i}$ | アイテム $i$ の重さ |\n",
+    "| $W$ | ナップサックの耐荷重 |\n",
+    "\n",
+    "この数理モデルの係数 $v_{i}, w_{i}, W$ に、次のインスタンスデータを入力して得られるインスタンスをソルバーで解くことを考えます：\n",
+    "\n",
+    "\n",
+    "| 文字 | 説明 |\n",
```

Comment:
```suggestion
    "| 文字 | 値 |\n",
```

Reply:
fix in https://github.com/Jij-Inc/JijModeling-Tutorials/pull/6/commits/343bcaac6e9f3dd1755281113e5e306433690be6